runtime/doc/quotes.txt

```diff
@@ -18,7 +18,7 @@ Coming with a very GUI mindset from Windows, I always thought of people using
 Vi as some kind of outer space alien in human clothes.  Once I tried I really
 got addicted by its power and now I found myself typing Vim keypresses in the
 oddest places! That's why I would like to see Vim embedded in every
-application which deals with text editing.  (Jos� Fonseca)
+application which deals with text editing.  (Jos� Fonseca)
```

Comment:
Did you change the encoding of the file?

It encoded in latin1 in Vim git repo.  I don't think that changing encoding should be done as part of this change.